frontend/src/redux/hostsSlice.ts

```diff
@@ -3,10 +3,11 @@ import { RootState } from "./store";
 import { fetchHosts } from "../common/api";
 import { Host, HostSummary } from "../common/types";
 
+//We need @Patryk Skupień here
```

Comment:
@skupien could you please check this file... below you are doing some weird stuff with ids assuming that they're integer values and it won't work after change to uuid